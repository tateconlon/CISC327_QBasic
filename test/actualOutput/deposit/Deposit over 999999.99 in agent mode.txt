login as agent successful
Account 1234567 does not exist
transaction code 999999999 invalid
Transaction summary written and system logged out successfully