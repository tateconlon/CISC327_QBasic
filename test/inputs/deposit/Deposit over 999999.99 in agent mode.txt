login
agent
deposit
1234567
999999999
logout