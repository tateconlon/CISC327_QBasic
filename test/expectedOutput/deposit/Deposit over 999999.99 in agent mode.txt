login as agent successful
Cannot deposit more than $999999.99 in a single transaction in agent mode
Transaction summary written and system logged out successfully